File: ExampleApp/app/src/main/java/com/motiv/example/UsersFragmentContract.java
package com.motiv.example;


public interface UsersFragmentContract {

    public interface View {

        public void usersListAdaptersetData(java.util.List<com.motiv.example.User> arg0);

        public void showToast(java.lang.String arg0);

        public void navigationControllerstartUserActivity(com.motiv.example.User arg0);
    }

    public interface Presenter {

        public void goApigetUsersList();

        public void eloonItemClick(int argument0, com.motiv.example.User argument1);
    }
}

File: ExampleApp/app/src/main/java/com/motiv/example/UsersFragmentContract.kt
 
package com.motiv.example
import kotlin.collections.List

public interface UsersFragmentContract {

    public interface View {

        public fun usersListAdaptersetData(arg0: List<com.motiv.example.User>): Unit
        public fun showToast(arg0: String): Unit
        public fun navigationControllerstartUserActivity(arg0: com.motiv.example.User): Unit
    }

    public interface Presenter {

        public fun goApigetUsersList(): Unit
        public fun eloonItemClick(argument0: Int, argument1: com.motiv.example.User): Unit
    }
}
